cli/command/load_balancer_test.go

```diff
@@ -89,6 +92,7 @@ func TestCreateLoadBalancer(t *testing.T) {
 
 	input := "l0 loadbalancer create "
 	input += "--private "
+	input += "--type " + string(config.DefaultLoadBalancerType) + " "
```

Comment:
It seems like using `config.DefaultLoadBalancerType`  just makes things less readable; why not use string literals for the test here? I don't see a reason to _have_ to use the default loadbalancer type variable?

Reply:
You definitely don't _have_ to use it. But all the usual pros about strongly typed apply here. It makes it harder for LoadBalancerType to be a random string.

Reply:
> It makes it harder for LoadBalancerType to be a random string

How could it be random if we are the ones setting it in the test?

Either way, consider this a [nit]